guide/14-deep-learning/text-classification-workflow.ipynb

```diff
@@ -0,0 +1,1222 @@
+{
```

Comment:
<ol><li><a href="https://huggingface.co/transformers/v3.0.2/pretrained_models.html" target="_blank">This page</a> mentions different <strong>transformer<s>s</s> </strong>architectures <a href="https://app.reviewnb.com/Esri/arcgis-python-api/pull/798/files/#References" target="_blank">[4]</a></li><li>The hyperlinks for references need to be relative, instead they point to absolute links on nbviewer</li><li>If your dataset consists of text in multiple languages you can <strong>choose<s>s</s></strong> models mentioned in the above link.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>fixed point 1 &amp; 3. For point 2, not sure why the hyper links are not working on nbviewer. I used the same code/procedure to create the hyper link as in the <a href="https://developers.arcgis.com/python/guide/how-named-entity-recognition-works/#Supported-formats-for-labeled-training-data" target="_blank">guide for entity recognition</a>. One can see how clicking to the <strong>Docanno [2]</strong> (last point) link will send the user the desired section on the web page. </p>